docker-compose.yml

```diff
@@ -1,17 +1,15 @@
 version: "3"
 services:
-  hmpps-probation-estate-api:
-    build:
-      context: .
-    network_mode: "host"
-    container_name: hmpps-probation-estate-api
+  postgres:
```

Comment:
Dunno if it is relevant but I usually put the postgres (and any other 3rd party tools) in a different docker compose - e.g. `docker-compose-test.yml`

Reply:
What would the other docker-compose file be used for in that case?

Reply:
You mean the docker compose for the app itself? That's a good question - I suspect it's supposed to be used to run the whole service locally without having to set up the debugging tool.

I wonder why is it in the template?